docs/blog/newsletter-june-2019.rst

```diff
@@ -0,0 +1,136 @@
+- Be specific in feedback requests. Instead of asking for general reviews of entire documents - vague requests can be overwhelming - try "Can you review these 5 code samples for syntax?" or "Are the use cases accurate? Am I missing any?" 
+-  Thoughtfulness pays: avoid phrasing your requests as demands. Consider your words as carefully for your coworkers as you do for your readers.
+- Highlight sections that you know need the most help: "I'm not confident about this section because I had to extrapolate from incomplete information. What did I get wrong here?"
+- Welcome all feedback with enthusiasm and gratitude. Use every opportunity to emphasize that good documentation is a team effort. And let people know when their feedback is incorporated, so they know they made a difference!
+
+As for tools and methods, it seems everyone has a different workflow!
+
+- Some use a `Headless CMS <https://headlesscms.org/>`_ or browser-based CMS like Paligo. Some tools have direct commenting features. Others create a Git PR from a draft, which opens up other commenting features (depending on the Git client).
+- `hypothesis.is <https://web.hypothes.is/>`_ allows commenting directly on web pages.
+- Dropbox Paper is useful for drafting content and granular commenting. It also exports to Markdown for publishing in other tools.
+- Some also collect feedback using Trello cards and Jira tickets.
+
+----------------------------
+Running objective interviews
+----------------------------
+
+Our Slack community enjoyed a lively discussion on how to maintain objectivity during interviews. Objective interviewing (and interviews in general!) is a topic that lends itself to extensive discussion, strong opinions, and many different perspectives.
+
+There's much more to unpack about this topic than we can cover in just one newsletter article, so here are some of this month's tips:
+
+- Recognize that interview methods are flawed and rife with bias, and work to do better.
+- The number of managers on a hiring panel matters less than the diversity of those managers, but both may help lessen bias on the panel.
+- If you can, consider introducing (as far as possible) a "blinded" interview process.
+- Consider training on how to be more aware of your biases.
+- Ask the questions you want the answers to.
+- Consider the subtext of questions that you do ask.
+- Exercise-based interviews can provide more relevant interactions with candidates than resumes and curated writing samples.
+
+Documentarians recommended some reading related to this topic:
+
+- `reWork Guide: Use structure and criteria <https://rework.withgoogle.com/print/guides/5443632811212800/>`__
+- `Should You Trust Your Guy in Hiring Decisions <https://www.shrm.org/ResourcesAndTools/hr-topics/talent-acquisition/Pages/Trust-Your-Gut-Hiring-Decisions.aspx>`__
```

Comment:
The link text should be "Gut" rather than "Guy," right?

Reply:
oh dear!